src/Util/Common.php

```diff
@@ -52,15 +52,57 @@ public static function isPharFile($path)
 
 
     /**
+     * Get absolute path, do not check exsit
+     *
+     * @param string $path The path to use.
+     *
+     * @return mixed
+     */
+    public static function absolutePath($path)
+    {
+        // Don't prefix absolute paths.
+        if (substr($path, 0, 1) !== '/') {
+            $path = getcwd().'/'.$path;
```

Comment:
getcwd will do a realpath (which might not be what you want according to the issue). You perhaps want to optionally inject that value as a parameter (which would allow to process on `getenv("PWD")` or some other OS-specific strategy to obtain the working directory).